src/base64/base64_functions.cpp

```diff
@@ -1,12 +1,13 @@
+#include "base64/base64_functions.h"
+#include <bitset>
 #include <cctype>
 #include <iomanip>
 #include <sstream>
 #include <string>
-#include <bitset>
-#include "base64/base64_functions.h"
 
 namespace cuhksz {
-static const std::string codes = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/=";
+static const char codes[] =
+    "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/=";
```

Comment:
This modification can save the static memory space and remove the needs of exit-time destructing